spiffworkflow-backend/src/spiffworkflow_backend/scripts/extract_from_task_data.py

```
@@ -0,0 +1,44 @@
+
+
+class ExtractFromTaskData(Script):
+    @staticmethod
+    def requires_privileged_permissions() -> bool:
+        """We have deemed this function safe to run without elevated permissions."""
+        return False
+
+    def get_description(self) -> str:
+        return "Removes variables from task data based on an optional predicate and returns them as a dictionary."
+
+    def run(self, script_attributes_context: ScriptAttributesContext, *args: Any, **kwargs: Any) -> Any:
```

Pylint (3.3.7)

[refactor] Either all return statements in a function should return an expression, or none of them should.

(R1710)